torchvision/transforms/transforms.py

```diff
@@ -2162,8 +2162,8 @@ def forward(self, tensor: Tensor) -> Tensor:
         Returns:
             PIL Image or Tensor: Transformed image.
         """
-        size = F.get_image_size(tensor)[::-1]
-        displacement = self.get_params(self.alpha, self.sigma, size)
+        _, height, width = F.get_dimensions(tensor)
+        displacement = self.get_params(self.alpha, self.sigma, [height, width])
```

Comment:
I don't know if this works with JIT, but maybe we can use some syntax sugar

```suggestion
        _, *size = F.get_dimensions(tensor)
        displacement = self.get_params(self.alpha, self.sigma, size)
```

Reply:
I don't think this is supported by JIT. We also don't have a JIT test for this transform so I don't wanna keep adding stuff without proper testing. Is that OK?